### PR TITLE
cronjob hdr v1

### PR DESCRIPTION
**Pull request by mdsupport**
Standard checks before running a script as cronjob.

Including batchcom script just to illustrate use of this library.

### CODE REVIEW

**Comment by bradymiller** [comment]
very neat!
Does the batchcom script work with this?

**Comment by mdsupport** [comment]
In current code, all users can run any script with $ignoreauth=true in the beginning without logging in.  Maybe that is by design but we needed that to work only for cli or cronjobs.

For batchom script only testing done was to make sure $ignoreauth is set when running in cli mode but not in online mode.  That part works.  

There is some error at the end but it is also seen in existing code.

File: library/allow_cronjobs.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?php
```
Comment (bradymiller) [inline library/allow_cronjobs.php]: this needs a author/license/copyright header. see here for example:
http://www.open-emr.org/wiki/index.php/How_to_Document_Your_Code_Properly#File-Level_DocBlock

**Comment by bradymiller** [comment]
will bring this into the codebase. thanks for the contribution! -brady

**Pull request merged**